bfasst/design.py

```diff
@@ -23,15 +23,22 @@ class Design:
     "class holding paths and other metadata for a given design" ""
 
     def __init__(self, dir_path, experiment_dir):
-        if not dir_path.is_dir() and (paths.DESIGNS_PATH / dir_path).is_dir():
+        if not dir_path.is_dir() and dir_path.is_absolute():
+            dir_path = dir_path
+        elif not dir_path.is_dir() and (paths.DESIGNS_PATH / dir_path).is_dir():
             dir_path = paths.DESIGNS_PATH / dir_path
-        dir_path = dir_path.absolute()
+
+        dir_path = paths.ROOT_PATH / dir_path.absolute()
```

Comment:
This doesn't look correct.  The right side of a path "/" operation should not be an absolute path.  What are you trying to accomplish with this line?

Reply:
reverted